### DISCUSSION TITLE
Add PNPM support to create-tina-app

### DISCUSSION THREAD

**Discussion by SrGeneroso** [tinacms/tinacms#4241] [Ideas]
I would like to use pnpm to install tina and in fact you can execute the script with pnpm, but then later you are asked to use yarn or npm. 🥺 That broke my heart 💔
I guess Tina devs are yarn users, anyways, cool app. I'll try to see if I'm good enough to add support to pnpm myself and PR it. Honestly is a minor problem, I can live with it. Maybe I should try yarn? In the meantime, I leave this here in case it has more supporters.

**Comment by jamespohalloran** [comment]
Tina works fine with pnpm (we actually use pnpm internally in our monorepo). Just run `pnpm install` in the project and delete the package-lock or yarn.lock file.
